contrib/podmanimage/stable/containers.conf

```diff
@@ -5,6 +5,7 @@ ipcns="host"
 utsns="host"
 cgroupns="host"
 cgroups="disabled"
+log_driver = "k8s_file"
```

Comment:
Is this change intentional?

Reply:
Yes it is a separate PR, Could add it as a fully separate from this, but I figured I would sneak it in while I was thinking about it.